mmv1/third_party/terraform/tests/resource_redis_instance_test.go

```diff
@@ -240,6 +240,8 @@ resource "google_redis_instance" "test" {
     notify-keyspace-events = ""
   }
   redis_version = "REDIS_5_0"
+	read_replicas_mode = "READ_REPLICAS_ENABLED"
```

Comment:
The indentation here seems to use tabs - should be spaces.

Reply:
fixed in https://github.com/GoogleCloudPlatform/magic-modules/pull/5793